File: src/Python/557.ReverseStringIII.py

class Solution:
    def reverseWords(self, s):
        """
        :type s: str
        :rtype: str
        """
        return ' '.join(map(lambda c : c[::-1], s.split(' ')))
            
if __name__ == '__main__':
    solution = Solution()
    print(solution.reverseWords("Let's take LeetCode contest"))
else:
    pass

File: src/Java/501-600/557.ReverseStringIII.java


import java.awt.List;
import java.util.ArrayList;
import java.util.Arrays;
import java.util.Collections;
import java.util.Comparator;
import java.util.LinkedList;
import java.util.Queue;
import java.util.HashMap;
import java.util.HashSet;

class Solution {
    public String reverseWords(String s) {
        String[] strs = s.split(" ");
        for (int i = 0; i < strs.length; ++i){
            strs[i] = new StringBuilder(strs[i]).reverse().toString();  
        }
        return String.join(" ", strs);
    }
}
